heltour/tournament/views.py

```diff
@@ -1839,14 +1839,20 @@ def view(self, post=False):
             season_player_set = SeasonPlayer.objects.filter(player__in=player_list, season=self.season).nocache()
             has_red_card_dict = {sp.player : sp.card_color == "red" for sp in season_player_set}
             
+            if self.league.competitor_type == 'team':
```

Comment:
We use this kind of if statement twice in two different places, probably a candidate for refactoring. 

```python
def is_team_league(self):
    ...
``` or something.

Reply:
I defined that function and and replaced those competitor_type=='team' calls, but I limited myself to those files that I had to touch anyway (views.py and models.py)